web/layout_attributes/using_layout_attributes/my_horizontal_element.html

```diff
@@ -0,0 +1,12 @@
+<link rel="import" href="packages/polymer/polymer.html">
+<dom-module id="my-horizontal-element">
+  <template>
+    <section class="layout horizontal">
```

Comment:
Yes they are opt-in. I couldn't check it because I couldn't make these examples work because of dart-lang/polymer-dart#584